LEAF_Nexus/index.php

```diff
@@ -358,19 +360,19 @@ function customTemplate($tpl)
                                            'css/view_group.css', ));
 
         $indicatorArray = $indicators->getIndicator((int)$_GET['indicatorID']);
-        $indicatorArray = array_map('Leaf\XSSHelpers::sanitizeHTML', $indicatorArray);
+        $indicatorArray = array_map('App\Leaf\XSSHelpers::sanitizeHTML', $indicatorArray);
```

Comment:
This is a namespace and yes Leaf is in the namespace. The new directory structure, and therefore the new namespace, is app/Leaf/...

Reply:
On line 12 with the whole use, wouldnt this just be XSSHelpers? I think that might be what Raphael is asking.

Reply:
Ok that makes sense but in this case because it's a value being passed to array_map I believe the whole namespace needs to be sent. I tested without it and ran into errors and came to this result.